relay-event-derive/src/lib.rs

```diff
@@ -544,17 +524,16 @@ fn parse_field_attributes(
                                         rv.characters = Some(attr);
                                     }
                                     _ => {
-                                        panic!("Got non string literal for max_chars");
+                                        panic!("Got non string literal for {}", ident);
                                     }
                                 }
                             } else if ident == "max_chars" {
                                 match name_value.lit {
-                                    Lit::Str(litstr) => {
-                                        let attr = parse_max_chars(litstr.value().as_str());
-                                        rv.max_chars = Some(quote!(#attr));
+                                    Lit::Int(litint) => {
+                                        rv.max_chars = Some(quote!(#litint));
                                     }
                                     _ => {
-                                        panic!("Got non string literal for max_chars");
+                                        panic!("Got non usize literal for max_chars");
```

Comment:
nit

```suggestion
                                        panic!("Got non integer literal for max_chars");
```

Reply:
Applied in [6c5e0d7](https://github.com/getsentry/relay/pull/2887/commits/6c5e0d7923dc75ff1d53046fd1f5adb9cd2839a0).